Controllers/UserController.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+namespace Controllers;
+
+use Models\UserModel;
+use Models\ViewModel;
+
+class UserController
+{
+    private $userModel;
+    private $viewModel;
+
+    public function __construct()
+    {
+        $this->userModel = new UserModel();
+        $this->viewModel = new ViewModel();
+    }
+
+    public function loginAction()
+    {
+        if ($_SERVER['REQUEST_METHOD'] == 'POST' && isset($_POST)) {
+            $login = $_POST['login'];
+            $password = $_POST['password'];
+            $result = $this->userModel->login($login, $password);
+            switch ($result) {
+                case 1 :
+                    $this->viewModel->location('/');
+                    break;
+                case 2 :
+                    $errMsg = 'Мама пока не дала разрешение на вход!';
+                    $this->viewModel->render(null, $errMsg);
+                    break;
+                case 0 :
+                    $errMsg = 'Неверный логин или пароль!';
+                    $this->viewModel->render(null, $errMsg);
+            }
+        } else {
+            $errMsg = 'Простите, но что-то пошло не так. Попробуйте еще раз';
+            $this->viewModel->render(null, $errMsg);
+        }
+    }
+
+    public function logoutAction()
+    {
+        session_destroy();
```

Comment:
i don't think it's a good idea to destroy session every time when this action is triggered.
it can be triggered directly from browser by link **http://site/user/logout** even if user is not logged in. no need to destroy session data in this case. You need to check is user logged in and only in this case drop session data.